html/projects.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
```

Comment:
I would like to reference / cite the use of external software somewhere in the html file. If not in the header then someplace else. The github repository is the only link that I have seen for facetview2. If there's another URL that we can use then I'd be happy to use it instead. And even though I've never seen this before, it's rare that I look at the source of web pages unless I'm developing them, so I can't say whether this happens in other sites or not, which would be a reason not to include such a comment.

Is there any harm in including a comment like this? For those interested in the capability, wanting to learn more, whatever?